code/modules/clothing/suits/armor.dm

```diff
@@ -327,11 +327,12 @@
 	icon_state = "bulletproof_ironhammer"
 
 /obj/item/clothing/suit/armor/laserproof
-	name = "ablative armor vest"
+	name = "full ablative armor vest"
```

Comment:
Denied. You can get gloves/boots

Reply:
It's an eris port but I assume it's because of the ablative gloves/boots from greyson right?

Reply:
Iron lock so ya graysons, but every cargopack/disk/kit has full sets of laser proofing.
only time you dont get full set is if you rng role